Configuration.props

```diff
@@ -151,8 +155,33 @@
     <AndroidSupportedTargetJitAbisForConditionalChecks Condition=" !$(AndroidSupportedTargetJitAbisForConditionalChecks.StartsWith (':')) " >:$(AndroidSupportedTargetJitAbisForConditionalChecks)</AndroidSupportedTargetJitAbisForConditionalChecks>
     <AndroidSupportedTargetJitAbisSplit>$(AndroidSupportedTargetJitAbis.Split(':'))</AndroidSupportedTargetJitAbisSplit>
   </PropertyGroup>
+
   <ItemGroup>
-    <AndroidSupportedTargetJitAbi Include="$(AndroidSupportedTargetJitAbisSplit)" />
+    <!-- It appears we cannot use batching outside a <Target> so we cannot use a set of items specifying the necessary API level to generate the
+         AndroidSupportedTargetJitAbi items with appropriate metadata. Thus we need to do it manually as below -->
+    <AndroidSupportedTargetJitAbi
+        Include="armeabi-v7a"
+        Condition=" $(AndroidSupportedTargetJitAbis.Contains ('armeabi-v7a')) ">
+      <ApiLevel>$(AndroidNdkApiLevel_ArmV7a)</ApiLevel>
+    </AndroidSupportedTargetJitAbi>
+
+    <AndroidSupportedTargetJitAbi
+        Include="arm64-v8a"
+        Condition=" $(AndroidSupportedTargetJitAbis.Contains ('arm64-v8a')) ">
+      <ApiLevel>$(AndroidNdkApiLevel_ArmV8a)</ApiLevel>
+    </AndroidSupportedTargetJitAbi>
+
+    <AndroidSupportedTargetJitAbi
+        Include="x86"
+        Condition=" $(AndroidSupportedTargetJitAbis.Contains ('x86')) ">
+      <ApiLevel>$(AndroidNdkApiLevel_X86)</ApiLevel>
+    </AndroidSupportedTargetJitAbi>
+
+    <AndroidSupportedTargetJitAbi
+        Include="x86_64"
+        Condition=" $(AndroidSupportedTargetJitAbis.Contains ('x86_64')) ">
```

Comment:
This should be `$(AndroidSupportedTargetJitAbisForConditionalChecks.Contains (':x86_64:'))`.